test/integration/etcd_storage_path_test.go

```diff
@@ -201,6 +197,7 @@ var kindWhiteList = sets.NewString(
 	"ImageStreamTag",
 	"UserIdentityMapping",
 	"ClusterResourceQuota",
+	"RoleBindingRestriction",
```

Comment:
i forgot, but please add explanation why we move these there (served using CRD)

Reply:
added

Reply:
Why are we removing the test data instead of just changing the location to match the new spot in ETCD?

Reply:
this change is now here: https://github.com/openshift/origin/pull/22699